bindings/wasm/examples/public/examples.js

```diff
@@ -259,12 +263,8 @@ export const examples = {
         const m = linearSegments > 2 ? linearSegments :
                                        n * q * majorRadius / threadRadius;
 
-        const circle = [];
-        const dPhi = 2 * 3.14159 / n;
-        const offset = 2;
-        for (let i = 0; i < n; ++i) {
-          circle.push([Math.cos(dPhi * i) + offset, Math.sin(dPhi * i)]);
-        }
+        const offset = 2.
+        const circle = CrossSection.circle(1., n).translate([offset, 0.]);
```

Comment:
No need for `.` in JS - everything is a double - even integers!

Reply:
Ah ocaml muscle memory acting up again.